dockerfiles/ubuntu/is-analytics/dashboard/Dockerfile

```diff
@@ -16,10 +16,43 @@
 #
 # ------------------------------------------------------------------------
 
-# set base Docker image to AdoptOpenJDK Ubuntu Docker image
-FROM adoptopenjdk:8u242-b08-jdk-hotspot-bionic
+# set base Docker image to JDK Ubuntu 20.04 Docker image
+FROM ubuntu:20.04
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
-      com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.7.0.4"
+      com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.7.0.5"
+
+#Install Open JDK
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends tzdata curl ca-certificates fontconfig locales \
+  && echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen \
+  && locale-gen en_US.UTF-8 \
+  && rm -rf /var/lib/apt/lists/*
+
+ENV JAVA_VERSION jdk8u292-b10
+
+RUN set -eux; \
+    ARCH="$(dpkg --print-architecture)"; \
+    case "${ARCH}" in \
+       amd64|x86_64) \
+         ESUM='699981083983b60a7eeb511ad640fae3ae4b879de5a3980fe837e8ade9c34a08'; \
+         BINARY_URL='https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u312-b07/OpenJDK8U-jdk_x64_linux_hotspot_8u312b07.tar.gz'; \
```

Comment:
Any reason not to update the latest version? In 5.10.0, we are using `jdk8u322-b06` [1]. There could be newer versions available as well

[1] https://github.com/wso2/docker-is/blob/f08f4027525fbc6562abd983f4fb491ff3ec3928/dockerfiles/jdk8/ubuntu/is/Dockerfile#L41

Reply:
Fixed from https://github.com/wso2/docker-is/pull/360/commits/f59fbfbe1d17ce24dbb89263706d00b3720ec4b2